src/buffer/out/textBuffer.cpp

```diff
@@ -1237,40 +1247,34 @@ const COORD TextBuffer::GetWordEnd(const COORD target, const std::wstring_view w
 // - The COORD for the first character of the next readable "word". If no next word, return one past the end of the buffer
 const COORD TextBuffer::_GetWordEndForAccessibility(const COORD target, const std::wstring_view wordDelimiters, const COORD lastCharPos) const
 {
-    const auto bufferSize = GetSize();
-    COORD result = target;
+    const auto bufferSize{ GetSize() };
+    COORD result{ target };
 
-    // Check if we're already on/past the last RegularChar
-    if (bufferSize.CompareInBounds(result, lastCharPos, true) >= 0)
+    if (bufferSize.CompareInBounds(target, lastCharPos, true) >= 0)
     {
-        return bufferSize.EndExclusive();
-    }
+        // if we're already on/past the last RegularChar,

```

Comment:
Not quite the last regular char. We need (user-entered) spaces, just not the extraneous thousands of newlines.